core/src/toga/app.py

```diff
@@ -349,6 +348,27 @@ def __init__(
     def _create_impl(self):
         return self.factory.App(interface=self)
 
+    @property
+    def paths(self):
+        """
+        Paths for known safe system locations. Using paths makes sure you have a consistent and known location to store
+        files for your app. Also, some platforms do not allow arbitrary file access to any location on disk. On Android,
+        there are many limitations when it comes to accessing file locations and the apis you can use to read from those
+        locations.
```

Comment:
Line length should be wrapped at ~80 chars. This should also mention that the locations returned are Pathlib objects.